configs/setups/awiesm3/awiesm3.yaml

```
@@ -0,0 +1,782 @@
+        execution_command: ${executable} -parallel ${runscript_name}
+        
+        veg_atm_fields: [GUE_LLAI, GUE_HLAI, GUE_FRAL, GUE_FRAH, GUE_TYPH, GUE_TYPL, T2M_LPJG, TPRE_LPJ, SNOC_LPJ, SNOD_LPJ, ST1L_LPJ, ST2L_LPJ, ST3L_LPJ, ST4L_LPJ, SM1L_LPJ, SM2L_LPJ, SM3L_LPJ, SM4L_LPJ, SWNR_LPJ, LWNR_LPJ, SHUM_LPJ, SPRE_LPJ, WSPD_LPJ, TMIN_LPJ, TMAX_LPJ]
+
+        parallel_file_movements: True
+        coupling_fields:
+                "[[veg_atm_fields-->FIELD]]":
+                        grid: ${lpj_guess.mygrid}-land
+
+        resolution: ${oifs.resolution}
+        grids:
+                lpjg:
+                        name: lpjg
+                        nx: "${nx}"
+                        ny: "${ny}" 
+                        oasis_grid_type: "D"
+                        oyac_grid_type: "LL"
+                        oyac_io_per_node: 4
+
+                TL159-land:
+                        name: TL159-land
+                        nx: "${nx}"
+                        ny: "${ny}" 
+                        oasis_grid_type: "D"
+                        oyac_grid_type: "LL"
+                        oyac_io_per_node: 4
+                TL255-land:
+                        name: TL255-land
+                        nx: "${nx}"
+                        ny: "${ny}" 
+                        oasis_grid_type: "D"
+                        oyac_grid_type: "LL"
+                        oyac_io_per_node: 4
+
+        choose_general.major_version:
+                "v3.4":
+                        version: "4.1.2"
+        
+        #proc_list_entry: '[str(x) for x in range(1,${nproc}+1)]'
+        nproc: 128
+        proc_list_entry: '[str(x) for x in range(1,${nproc}+1)]' 
+        proc_list: $(( ${proc_list_entry} ))
+
+        run_year: ${start_date!syear}
+        run_month: 1
+        run_day: 1
+        runtime_seconds: "$(( ${next_date} - ${start_date} ))"
+        ndays_per_run: "$(( ${runtime_seconds} / 86400 ))"
+        TM5: 0
+        fourxCO2: f
+        onepcCO2: f
+        fixed_CO2: -1
+        fixed_ndep: -1
+        fixed_LU: -1
+   
+        forcing_sources:                
+                soil_layer_type: ${lpj_guess.input_dir}/slt/slt_${mygrid}.nc
+                tmp_rc_file: ${lpj_guess.input_dir}/lpjg_steps.rc.j2
+                CO2: ${lpj_guess.input_dir}/cmip6_forcing/mole_fraction_of_carbon_dioxide_in_air_input4MIPs_lpjg.nc
+                dir_landuse: ${lpj_guess.input_dir}/dirmaker
+                dir_fire: ${lpj_guess.input_dir}/dirmaker
+                ins_files: ${namelist_dir}/*.ins
+                #ece_gridlist: ${lpj_guess.input_dir}/gridlists/gridlist_TL159_CORE2.txt #is this the correct file for ece_gridlist.txt?
+                ece_gridlist: ${lpj_guess.input_dir}/gridlists/gridlist_TL255_CORE2.txt #is this the correct file for ece_gridlist.txt? 
+                lpjg_cmip6_output_ins: ${namelist_dir}/lpjg_default_cmip6_output.ins
+                "all_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/*.ins 
```

This is strange. You want to use the syntax `[[proc_list-->PROC_NUM]]` when you need to expand a string with a list. For that you use `PROC_NUM` on the right hand side as the placeholder that takes the values of the list as the string is expanded.

But here your right hand side has not PROC_NUM. I think what this is doing is to do the same operation of copying of all files globbed a number of PROC_NUM times more than what you actually need. You probably want to change this (and all the ones below and in the `forcing_in_work`) to:

```suggestion
                "all_ins": ${namelist_dir}/*.ins 
```

@hajlaci @JanStreffing

---

I'm not convinced we can get rid of this. We need the PROC_NUM of the forcing_in_work side:

`"all_ins_[[proc_list-->PROC_NUM]]": runPROC_NUM/*.ins`

And it seems to have to be part of the name of the esm_tools variable. Which suggests to me, that we should already set in PROC_NUM times on the forcing_sources side:

"all_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/*.ins

---

you are 100% right @JanStreffing, what a explosion of identical files... Okay, it can stay as it is. This is not ideal. The amount of files this thing will produce will increase with the number of mpi tasks and the number of chunks one runs. I think this can result into troubles in the future, but I'm sure you are aware about that.

---

@mandresm needs to open an issue before resolving it